tests/cephfs/bug-1980920.py

```diff
@@ -111,6 +111,9 @@ def run(ceph_cluster, **kw):
                 fs_util.remove_subvolume(
                     client1, "cephfs", new_subvolume_name, force=True
                 )
```

Comment:
Also here we need to check if clone status is not in "canceled" state

Reply:
Updated the comments and re ran the test 
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SMUA5X/